internal/host/host.go

```diff
@@ -586,8 +588,13 @@ func (m *Manager) UpdateInstallProgress(ctx context.Context, h *models.Host, pro
 			swag.StringValue(h.Status), models.HostStatusError, statusInfo)
 	case models.HostStageRebooting:
 		if swag.StringValue(h.Kind) == models.HostKindAddToExistingClusterHost {
+			statusInfoMessage := statusInfoRebootingDay2
+			if m.kubeApiEnabled {
+				statusInfoMessage = ""
```

Comment:
Why is this needed? Just add an `&&` to the `if swag.StringValue(h.Kind) == models.HostKindAddToExistingClusterHost` condition above

Reply:
A few lines below we update the host with a status, this update does not depend on KubeAPI being enabled. 

I don't think it's appropriate to add the condition to the `if` statement for this reason.